app/models/item_change_set.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Represents a set of changes to an item.
```

Comment:
Is this intended only for updating Cocina::Models::DRO (i.e., not APOs or Collections)? If so, can you make it explicit?

Reply:
I'm not sure what terminology is explicit in Argo.  I thought "item" was explicit.
<img width="415" alt="Screen Shot 2021-03-23 at 5 42 26 PM" src="https://user-images.githubusercontent.com/92044/112228331-25153400-8bff-11eb-921c-5aea3db35dc6.png">